.goreleaser.yml

```diff
@@ -262,16 +262,29 @@ release:
   #  - glob: ./glob/foo/to/bar/file/foobar/override_from_previous
 
 blobs:
-  -
-    provider: s3
-    region: us-east-1
-    bucket: '{{ .Env.AWS_S3_BUCKET }}'
+  - provider: s3
+    disable: 'false'
     ids:
       - s3-versioned
-      - s3-unversioned
+    bucket: '{{ .Env.AWS_S3_BUCKET }}'
+    region: us-east-1
+    directory: '/'
     acl: public-read
-    disable: '{{ ne .Prerelease "" }}'
+    extra_files:
+      - glob: ./dist/s3-versioned_*/**
+    extra_files_only: true
 
+  - provider: s3
+    disable: '{{ if .Prerelease }}true{{ else }}false{{ end }}'
+    ids:
+      - s3-unversioned
```

Comment:
Ok, for now, I think we are building the same binaries twice, we could use `name_template` in the `extra_files` to change the name of the s3_versioned files.